### PR TITLE
drop python < 3.6 for opencensus-ext-stackdriver

### PR DESCRIPTION
**Pull request by aabmass**
Drop support for Python versions < 3.6 (breaking change) for `opencensus-ext-stackdriver`. This is in preparation for upgrading to the `google-cloud-monitoring ~= 2.0`

- Update setup.py classifiers and python_requires
- Update tox.ini to only include this package in compatible python version builds

### CODE REVIEW

File: contrib/opencensus-ext-stackdriver/CHANGELOG.md [hunk 1]
```diff
@@ -2,30 +2,33 @@
 
 ## Unreleased
 
+- Drop python <3.6 in opencensus-ext-stackdriver
+  [#1050](https://github.com/census-instrumentation/opencensus-python/pull/1050)
```
Comment (lzchen) [inline contrib/opencensus-ext-stackdriver/CHANGELOG.md:6]:
```suggestion
  [#1056](https://github.com/census-instrumentation/opencensus-python/pull/1056)
```

**Review by lzchen — APPROVED** [review]
Is the next release going to bump the minor version?

**Comment by aabmass** [comment]
> Is the next release going to bump the minor version?

Yes that would be ideal. Are we able to version these packages separately or you want to keep them all in lockstep?

**Comment by aabmass** [comment]
@googlebot I fixed it.

**Comment by lzchen** [comment]
@aabmass 
[Azure](https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-azure/opencensus/ext/azure/common/version.py#L15) is already not in lockstep so I'm totally fine in bumping the stackdriver version. As well, we release stuff manually too (no script) so no advantage in having them all the same version either.

**Review by tbarker25 — APPROVED** [review]

**Comment by aabmass** [comment]
@googlebot rescan

**Comment by mortendaehli** [comment]
@aabmass, this made the nox test fail for Python 2.7 and 3.5. How should this be handled since this is only an extention?

**Pull request merged**